src/object-properties.js

```diff
@@ -132,7 +132,8 @@ const properties = {
  * @typedef {object} NavigationAction
  * @property {'nextSheet'|'prevSheet'|'firstSheet'|'lastSheet'|'goToSheet'|'goToSheetById'|'goToSheetById'|'goToStory'|'openWebsite'|'odagLink'|'openChainedApp'} action - Name of navigation action
  * @property {string=} appId - app ID. Required for 'openChainedApp'
- * @property {string=} sheet - sheet ID. Required for 'goToSheet' and 'goToSheetById'
+ * @property {string=} sheet - sheet - sheet ID. Required for 'goToSheet' and 'goToSheetById'
+ * @property {string=} chart - chart - chart ID. Required for 'goToSheet' and specify the chart
```

Comment:
I don't think it is required for goToSheet right? you can still just go to a sheet and the if you want to you specify a chart as well? Also looks like a typo, it says chart - chart, and sheet - sheet

Reply:
I saw the other items from navigations are all added so added this one as well. 
Sorry for my messy PR, should clean that up before put it for review. All comments are fixed.

Reply:
I mean you will need to add it here, but the chart prop should not say that it is required. For now though I think it is good that we don't change the API spec, you can do that in a different PR